app/view/main/Map.js

```diff
@@ -48,6 +48,11 @@ Ext.define('CpsiMapview.view.main.Map', {
             listeners: {
                 afterrender: 'initializeMeasureBtn'
             }
+        }, {
+            xtype: 'cmv_timeslider',
+            startDate: new Date(1998, 0, 1),
+            endDate: new Date(Date.now()),
+            layerNames: [] // TODO: fill me
```

Comment:
I think the slider can simply guess these by iteration over all layers (util), and seeing if a property is set.

Reply:
Alright, I'll introduce such an util this in an upcoming PR